docs/getting-started/local_instance.md

```diff
@@ -32,9 +32,11 @@ python butler.py run_server
 This may take a few seconds to start. Once you see an output line like
 `INFO <timestamp> admin_server.py:<num>] Starting admin server`, you should be
 able to navigate to [http://localhost:9000](http://localhost:9000) to view the
-web interface. Note that the local instance may use ports other than 9000, such
-as 9008, for things like uploading files. Therefore, using port forwarding to
-access a local instance may break.
+web interface.
+Note: the local instance may use ports [other than 9000](https://github.com/google/clusterfuzz/blob/master/src/local/butler/constants.py),
+such as 9008, for things like uploading files. Therefore, using the local
```

Comment:
How about generalize this, ask users to tweak network rules like firewall and port forwarding.

Reply:
I think mentioning port forwarding and allowing through firewall is kind of too specific and I don't want to complicate the docs for the majority of users for whom this isn't an issue. What do you think of changing the last sentence to this:
>Therefore, using the local instance may break if the needed ports are unavailable or if you can only access some of the needed ports from your browser.

Reply:
sgtm

Reply:
I ended up warning about port forwarding and firewall